project/Dependencies.scala

```diff
@@ -37,19 +37,21 @@ object Dependencies {
   val scalaTestVersion = "3.1.4"
   val scalaCheckVersion = "1.15.1"
 
+    def getScalaVersion() = {
+    // don't allow full override to keep compatible with the version of silencer
+    // don't mandate patch not specified to allow builds to migrate
+    System.getProperty("akka.build.scalaVersion", "default") match {
+      case twoThirteen if twoThirteen.startsWith("2.13") => scala213Version
```

Comment:
(a bit weird but it was already like this so :+1: :) )

Reply:
I can include any encoding you prefer, I found it weird too :)

Reply:
I suspect we still have jenkins jobs that pass `2.13.x` as `akka.build.scalaVersion`, and we wanted to fix those before changing the `startsWith` match to an exact match. Might be best to cover in another PR?

Reply:
deal 👍